File: eform-client/src/app/plugins/modules/time-planning-pn/models/pay-rule-sets/pay-rule-set-create.model.ts
import { PayDayRuleModel } from './pay-day-rule.model';
import { PayDayTypeRuleModel } from '../pay-day-type-rules/pay-day-type-rule.model';

export class PayRuleSetCreateModel {
  name: string;
  payDayRules: PayDayRuleModel[];
  payDayTypeRules: PayDayTypeRuleModel[];
}

File: eFormAPI/Plugins/TimePlanning.Pn/TimePlanning.Pn/Infrastructure/Models/PayRuleSet/PayRuleSetCreateModel.cs
/*
The MIT License (MIT)

Copyright (c) 2007 - 2021 Microting A/S
*/

namespace TimePlanning.Pn.Infrastructure.Models.PayRuleSet;

using System.Collections.Generic;
using PayDayTypeRule;

public class PayRuleSetCreateModel
{
    public string Name { get; set; }
    public List<PayDayRuleModel> PayDayRules { get; set; } = new List<PayDayRuleModel>();
    public List<PayDayTypeRuleModel> PayDayTypeRules { get; set; } = new List<PayDayTypeRuleModel>();
}